apps/mobile/src/libs/components/background-wrapper/background-wrapper.tsx

```diff
@@ -0,0 +1,47 @@
+import { ReactNode } from "react";
+import { type ImageSourcePropType } from "react-native";
+
+import { Image, View } from "~/libs/components/components";
+import { globalStyles } from "~/libs/styles/styles";
+
+import { Planet } from "./libs/components/planet/planet";
+import { styles } from "./styles";
+
+const circleTop =
+	require("~/assets/images/top-circle.png") as ImageSourcePropType;
+const circleBottom =
+	require("~/assets/images/bottom-circle.png") as ImageSourcePropType;
+
+type Properties = {
+	children: ReactNode;
+};
+
+const BackgroundWrapper = ({ children }: Properties) => {
+	return (
+		<View style={[styles.container, globalStyles.flex1]}>
```

Comment:
It's just a nitpick, but let's use globalStyles first and after them the custom ones.
```suggestion
		<View style={[globalStyles.flex1, styles.container]}>
```